<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PdPageGen</class>
 <widget class="QDialog" name="PdPageGen">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generate Index Page for Directory</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="layTitle">
       <item>
        <widget class="QLabel" name="lblTitle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Tit&amp;le for Index:-</string>
         </property>
         <property name="buddy">
          <cstring>txtTitle</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtTitle">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="layDesc">
       <item>
        <widget class="QLabel" name="lblDesc">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Descriptio&amp;n of directory content:-</string>
         </property>
         <property name="buddy">
          <cstring>txtDesc</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PicDokPlainTextEdit" name="txtDesc">
         <property name="toolTip">
          <string>Enter text here.</string>
         </property>
         <property name="tabChangesFocus">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="laySubject">
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QLabel" name="lblTopic">
         <property name="text">
          <string>&amp;Main subjects in directory</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>lstSubject</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PicDokListView" name="lstSubject">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="movement">
          <enum>QListView::Free</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layAddDel">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="AddDelSpa1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbAdd">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>128</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>128</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>137</red>
                 <green>135</green>
                 <blue>133</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Add line (Ctrl + )</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
           <property name="shortcut">
            <string>Ctrl++</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>137</red>
                 <green>135</green>
                 <blue>133</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="toolTip">
            <string>Delete line (Ctrl - )</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+-</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="addDelSpa2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horSubSpa1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="layOfile">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="lblOfile">
         <property name="text">
          <string>Output File Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtOfile">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="cbPreview">
         <property name="text">
          <string>&amp;Preview</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horOfileSpa">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="layButtons">
       <item>
        <widget class="QPushButton" name="pbQuit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="shortcut">
          <string>Esc</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horButSpa1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbClose">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>Save &amp;&amp; &amp;Close</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+W</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horButSpa2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbSave">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Save</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+S</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horButSpa3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbGen">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>28</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Generate Index</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PicDokPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>picdokplaintextedit.h</header>
  </customwidget>
  <customwidget>
   <class>PicDokListView</class>
   <extends>QListView</extends>
   <header>picdoklistview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtTitle</tabstop>
  <tabstop>txtDesc</tabstop>
  <tabstop>lstSubject</tabstop>
  <tabstop>txtOfile</tabstop>
  <tabstop>pbQuit</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbGen</tabstop>
  <tabstop>pbAdd</tabstop>
  <tabstop>pbDel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbSave</sender>
   <signal>clicked()</signal>
   <receiver>PdPageGen</receiver>
   <slot>doSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbGen</sender>
   <signal>clicked()</signal>
   <receiver>PdPageGen</receiver>
   <slot>doGen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>PdPageGen</receiver>
   <slot>doClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbQuit</sender>
   <signal>clicked()</signal>
   <receiver>PdPageGen</receiver>
   <slot>doQuit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>25</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbAdd</sender>
   <signal>clicked()</signal>
   <receiver>PdPageGen</receiver>
   <slot>doAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDel</sender>
   <signal>clicked()</signal>
   <receiver>PdPageGen</receiver>
   <slot>doDel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doQuit()</slot>
  <slot>doSave()</slot>
  <slot>doClose()</slot>
  <slot>doGen()</slot>
  <slot>doAdd()</slot>
  <slot>doDel()</slot>
 </slots>
</ui>
